notation.go

```diff
@@ -194,6 +185,24 @@ type VerificationOutcome struct {
 	Error error
 }
 
+func (outcome *VerificationOutcome) UserMetadata() (map[string]string, error) {
+	if outcome.EnvelopeContent == nil {
+		return map[string]string{}, nil
```

Comment:
If `outcome.EnvelopeContent` is nil, should we return the outcome error instead of an empty map?